src/components/fb/FidelityBondSteps.tsx

```diff
@@ -0,0 +1,294 @@
+        <div className={styles.utxoDetails}>
+          <div>{utxo.path}</div>
+          <div>&#183;</div>
+          <div>{utxo.confirmations} Confirmations</div>
+        </div>
+      </div>
+      {isLoading && (
+        <div className={styles.utxoLoadingSpinner}>
+          <rb.Spinner as="span" animation="border" size="sm" role="status" aria-hidden="true" />
+        </div>
+      )}
+      {!isLoading && utxo.frozen && !utxo.locktime && (
+        <div className={cx('utxoLabel', 'utxoFrozen')}>
+          <Sprite symbol="lock" width="18" height="18" />
+          <div>frozen</div>
+        </div>
+      )}
+      {!isLoading && utxo.locktime && (
+        <div className={cx('utxoLabel', 'utxoFidelityBond')}>
+          <Sprite symbol="lock" width="18" height="18" />
+          <div>locked</div>
+        </div>
+      )}
+      {!isLoading && !utxo.frozen && utxo.label === 'cj-out' && (
+        <div className={cx('utxoLabel', 'utxoCjOut')}>
+          <Sprite symbol="cj" width="18" height="18" />
+          <div>cj-out</div>
+        </div>
+      )}
+    </div>
+  )
+}
+
+const SelectUtxos = ({ jar, utxos, selectedUtxos, onUtxoSelected, onUtxoDeselected }: SelectUtxosProps) => {
+  return (
+    <div className="d-flex flex-column gap-4">
+      <div className={styles.stepDescription}>
+        Select one or more UTXOs from jar #{jar} to use for the fidelity bond.
+      </div>
+      {utxos.map((utxo, index) => (
+        <UtxoCard
+          key={index}
+          utxo={utxo}
+          isSelectable={!utxo.frozen}
+          isSelected={fb.utxo.isInList(utxo, selectedUtxos)}
+          onClick={() => {
+            if (fb.utxo.isInList(utxo, selectedUtxos)) {
+              onUtxoDeselected(utxo)
+            } else {
+              onUtxoSelected(utxo)
+            }
+          }}
+        />
+      ))}
+    </div>
+  )
+}
+
+const FreezeUtxos = ({ jar, utxos, selectedUtxos, isLoading = false }: FreezeUtxosProps) => {
+  return (
+    <div className="d-flex flex-column gap-4">
+      <div className={styles.stepDescription}>Selected UTXOs:</div>
+      {selectedUtxos.map((utxo, index) => (
+        <UtxoCard key={index} utxo={utxo} isSelectable={false} isSelected={true} />
+      ))}
+      <div className={styles.stepDescription}>
+        The following UTXOs will not be used for the fidelity bond. They will be frozen in order to remain in jar #{jar}
+        . You can unfreeze them anytime after creating the fidelity bond.
+      </div>
+      {fb.utxo.utxosToFreeze(utxos, selectedUtxos).map((utxo, index) => (
+        <UtxoCard key={index} utxo={utxo} isSelectable={false} isSelected={false} isLoading={isLoading} />
+      ))}
+    </div>
+  )
+}
+
+const ReviewInputs = ({ lockDate, jar, utxos, selectedUtxos, timelockedAddress }: ReviewInputsProps) => {
+  const settings = useSettings()
+
+  const UtxoSummary = ({ title, icon, utxos }: { title: string; icon: string; utxos: Array<Utxo> }) => {
+    const settings = useSettings()
```

Comment:
Should either `UtxoSummary` be moved out of `ReviewInputs` or the line `const settings = useSettings()` deleted?

Reply:
Removed it. ✅